testdata/project-v3-addon/config/manager/manager.yaml

```diff
@@ -22,6 +22,8 @@ spec:
       labels:
         control-plane: controller-manager
     spec:
+      securityContext:
+        runAsUser: 65532
       containers:
       - command:
```

Comment:
Could we add in the manifests `allowPrivilegeEscalation: false`?

Reply:
Done